src/components/CommentPostingForm.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState, useEffect } from "react";
+import postArticleComment from "../utils/postFunctions";
+import { useParams } from 'react-router-dom';
+
+function CommentPostingForm({commentList,setCommentList}){
+
+    const [err, setErr] = useState(null)
+    const [feedback,setFeedback]=useState(null)
+    const {article_id} = useParams()
+    const [comment,setComment]=useState({
+        author:"cooljmessy",
+        body:""
+    })
+    const [loading,setLoading] = useState(false)
+
+
+  function handleSubmit(event){
+    setLoading(true)
+    event.preventDefault()
+    if(comment.body.length > 2){
+        if(setLoading){
+            setFeedback("Attempting to post comment...")
+        }
+        postArticleComment(article_id,comment.author,comment.body)
+        .then(()=>{
+            setLoading(false)
+            setErr(null)
+            setFeedback("Comment posted successfully!")
+        })
+        .catch((err)=>{
+            setFeedback(null)
+            setErr('Something went wrong, please try again.')
+            setCommentList(commentList.slice(0,-1))
+        })
+        setCommentList([...commentList,comment])
+    } 
```

Comment:
I would have the input at the top of comments so the ability to post is easily accessed by our client